src/modules/iasql_functions/rpcs/iasql_help.ts

```diff
@@ -0,0 +1,65 @@
+import { IasqlFunctions } from '..';
+import {
+  Context,
+  PostTransactionCheck,
+  PreTransactionCheck,
+  RpcBase,
+  RpcResponseObject,
+} from '../../interfaces';
+
+/**
+ * Method to list IaSQL functions, their description and usage example
+ *
+ * Returns following columns:
+ * - name: function name
+ * - signature: signature used to call the function
+ * - description: a description on what the function does
+ * - sample_usage: sample usage of the function
+ *
+ */
+export class IasqlHelp extends RpcBase {
+  /**
+   * @internal
+   */
+  module: IasqlFunctions;
+  /** @internal */
+  preTransactionCheck = PreTransactionCheck.NO_CHECK;
+  /** @internal */
+  postTransactionCheck = PostTransactionCheck.NO_CHECK;
+  /** @internal */
+  inputTable = {};
+  /**
+   * @internal
+   */
+  outputTable = {
+    name: 'varchar',
+    signature: 'varchar',
+    description: 'varchar',
+    sample_usage: 'varchar',
+  } as const;
+  call = async (
+    dbId: string,
+    _dbUser: string,
+    _ctx: Context,
+  ): Promise<RpcResponseObject<typeof this.outputTable>[]> => {
+    const returnList = [];
+
+    for (const [functionName, documentation] of Object.entries(this.module.provides.functions)) {
+      if (!!documentation.description && !!documentation.sample_usage)
```

Comment:
Won't this conditional skip anything that doesn't have a description or sample usage? It may be worthwhile to have an `else` branch that does a `logger.debug` call so we can potentially notice this during tests.

Reply:
Changes in the new commit will take this into account.